docs/application/web/guides/personal/bookmarks.md

```diff
@@ -1,5 +1,9 @@
 # Bookmarks
 
+> **Note**
+>
+> Since Tizen 6.0, the Bookmark API is deprecated and using it is not recommended.
```

Comment:
Thanks for the update.

However, the note style we use in Tizen docs is changed.
Would you please check https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md and change the note style?

\> [!NOTE]
\>Since Tizen 6.0, the Bookmark API is deprecated and using it is not recommended.

Sorry for the late sharing this.

Reply:
I updated this, but it looks like it is not generated in the same way as presented in https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md
in file:
https://github.com/Samsung/tizen-docs/blob/6af9bd943ccf001cd691cc7a39977307f291035d/docs/application/web/guides/personal/bookmarks.md

Reply:
Even though, it looks good when checking on https://docs1.stg.tizen.org/staging/1077/application/web/guides/personal/bookmarks/

Reply:
We don't want people to use it anymore - that's why we are deprecating it, so maybe "Caution" would be more suitable - red color attracts attention more than blue.
What do you think about that?

Reply:
@pkosko As you said "View file" doesn't convert well this new style yet.
But as you also checked, it will be converted well on docs.tizen.org. So Don't need to worry about that.

@rwalczyna I think your suggestion to use "Caution" is good. "Caution" can be used instead of "Note".

Reply:
@mijin85cho @rwalczyna Done